infra/terraform/test-org/github/files.tf

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "github_repository_file" "renovate_json_tgm" {
+  for_each            = data.github_repository.repo
+  repository          = each.value.name
+  branch              = each.value.default_branch
+  file                = ".github/renovate.json"
+  commit_message      = "chore: update renovate.json"
+  commit_author       = "CFT Bot"
+  commit_email        = "cloud-foundation-bot@google.com"
```

Comment:
nit: might be good to pull these into a local since we may want to add more config files in the future.